Organizer-Timeline/1-Month-Before/Workshops.md

```diff
@@ -0,0 +1,32 @@
+# Finding Great Workshop Content at O(1) Speed
+
+As a hackathon organizer, you have to worry about getting enough food, figuring out the budget, setting the schedule, contacting sponsors, etc. Then you realize your hackathon doesn't have any workshops on its schedule or you don't know what workshops can appeal to your hackers, each with different experience levels. You want your workshop to be fun and educational. But your team is already busy managing everything else. The last thing you need is to worry about finding time to prepare slides, dig through API documentation, and setup a development environment.
```

Comment:
I like the intent here, but it feels like a sell for Localhost versus a general guide on how to find great workshop content. 

My suggestion: have a paragraph or two on how/what organizers need to do for finding workshops content, speakers, space, etc. Then, have a section on MLH Localhost that can help them solve all of the above issues. It feels like a much greater sell to me because you present to them the problem, then potential solution of LH.

Reply:
hey @mlh-henry let me know if you're going to make a change here; otherwise I'll merge.

Reply:
I'll make a quick change. Then I can merge it.